src/API/HTTPAdapter/CurlHTTPAdapter.php

```diff
@@ -35,7 +35,7 @@ protected function parseHeaders($headers)
 
     public function request(string $method, string $endpoint, array $params = [], array $headers = []): Response
     {
-        $curl = curl_init(sprintf($this->getBaseUrl(), $endpoint));
+        $curl = curl_init(sprintf('%1$s%2$s', $this->getBaseUrl(), $endpoint));
```

Comment:
I think no need for positional argument, we can just use `%s%s'` :pray:

Reply:
https://github.com/NextmediaMa/youcan-payment-php-sdk/pull/7/commits/a405ca770a72288323cecdca3609fb3af027fbd2